test/components/CalendarMonth_spec.jsx

```diff
@@ -46,5 +48,50 @@ describe('CalendarMonth', () => {
       expect(typeof isVisible).to.equal('boolean');
       expect(wrapper.find('#month-element').exists()).to.equal(true);
     });
+
+    describeIfWindow('setMonthTitleHeight', () => {
+      const awaitChange = fn => new Promise((resolve, reject) => {
```

Comment:
This seems very brittle; it’s explicitly racing against the change. These tests should find a way to explicitly wait for the proper time instead.